db/seeds.rb

```diff
@@ -5,3 +5,18 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+
+first_user = User.create([{name: "Zuhra"}])
+second_user = User.create([{name: "Hashimi"}])
+
+recipe1 = Recipe.create(name: "Banana Smoothie", preparationTime: 10, cookingTime: 0, description: "A delicious smoothie made with banana, milk and honey", public: true, user_id: 1)
+recipe2 = Recipe.create(name: "Apple Pie", preparationTime: 30, cookingTime: 60, description: "A delicious pie made with apple, flour, sugar and butter", public: true, user_id: 1)
+recipe3 = Recipe.create(name: "Orange Juice", preparationTime: 10, cookingTime: 0, description: "A delicious juice made with orange, sugar and water", public: true, user_id: 1)
+
+recipe_food = RecipeFood.create(quantity: 1, recipe_id: recipe1.id, food_id: banana.id)
+recipe_food2 = RecipeFood.create(quantity: 1, recipe_id: recipe2.id, food_id: apple.id)
+recipe_food3 = RecipeFood.create(quantity: 1, recipe_id: recipe3.id, food_id: orange.id)
+recipe_food4 = RecipeFood.create(quantity: 1, recipe_id: recipe1.id, food_id: apple.id)
+recipe_food5 = RecipeFood.create(quantity: 100, recipe_id: recipe2.id, food_id: onion.id)
```

Comment:
- [ ] there's an error while seeding the database , the variables banana , apple , orange etc ... are not defined ! kindly fix that issue  
![image](https://user-images.githubusercontent.com/89282221/230734848-212faa44-a9e1-471d-b6a4-39302fc249f7.png)